### ISSUE TITLE
not able to download using S3 torrent

### ISSUE THREAD

**Issue by srmitharwal** [webtorrent/webtorrent#1474]
**What version of WebTorrent?**

0.102.1
**What operating system and Node.js version?**

Ubuntu 17.04 Server, Nodejs 10.8.0

**What did you expect to happen?**
I want  to download the file using s3 torrent .

**What actually happened?**
I am trying to download the files which are in Amazon S3 using web torrent client. I am able to successfully generate torrents for them but not able to download.  File is getting download by using same S3 torrent if I use some other torrent client.

**Comment by DiegoRBaquero** [comment]
Are you using http://docs.aws.amazon.com/AmazonS3/latest/dev/S3Torrent.html ? See https://github.com/webtorrent/webtorrent/issues/979

**Comment by srmitharwal** [comment]
Yes, I am using the same approach. I went through the #979 but as mentioned in the amazon BitTorrent doc, we do not need to seed our public file. Amazon takes care of that , we just have to generate torrent and we can download it using any torrent client. I used peerflix to download the file, its working fine. I went through the webtorrent code, I figured out that it is able to identity the peers but not able to set up tcp connection. WebTorrent client is sending SYC to AWS but client is not receiving SYN-ACK from AWS.

**Comment by DiegoRBaquero** [comment]
Can anyone post an S3 torrent link? The torrent metadata points to the S3 file as a webseed?

**Comment by mistersdevelopment** [comment]
Here is an example link: https://aft-vbi-pds.s3.amazonaws.com/LICENSE?torrent. This is a license in a public data set in S3 (https://registry.opendata.aws/amazon-bin-imagery/). You can see the non-torrent version here: https://aft-vbi-pds.s3.amazonaws.com/LICENSE.

**Comment by balupton** [comment]
Just checking if there is anything new on this? Am I correct in interpretation that there is a bug or lack of support affecting webtorrent ability to download from the s3 torrent service?

my personal goal is to have the local webtorrent instance generate a torrent for a local media file, upload the media file to s3, amend the torrent data to point to the s3 torrent location as a peer, then store the torrent file in a cloud database for retrieval by others who will use webtorrent to get it from the original user as well as s3 if need be.
